### DISCUSSION TITLE
What about adding animated SVG support in Excalidraw?

### DISCUSSION THREAD

**Discussion by tofDou** [excalidraw/excalidraw#8721] [General]
Hello,

I know there is multiple way to animate SVG:
- build-in "simple" animation like this using SMIL animations (![infinite-spinner](https://github.com/user-attachments/assets/f287dd5d-2c78-4ea3-a3f9-58359770e850)
)
- more complex animation like this with integrated css (
![animated](https://github.com/user-attachments/assets/09c77051-fc70-4845-822a-cbc313d2b774)

)
- more complexe animation with integrated javascript ([animated2](https://github.com/user-attachments/assets/0b7f9380-ff97-4905-86c5-8a97193eb9db) ) not working in github but ok directly in a browser like chrome for exemple

This is very useful to be able to animate some shapes in a schema.
To add a complex mechanism to animate things in Excalidraw is not what it is requested but only great display of imported animated SVG!

Share your point of view and give me the tips if I miss something...

**Comment by Mrazator** [comment]
Hi, as of now, there is https://github.com/dai-shi/excalidraw-animate.

In the future, we might ship some animations to the excalidraw itself. Perhaps starting from more simple ones like those suggested here https://github.com/excalidraw/excalidraw/issues/8399.

**Reply by RoyiAvital** [reply]
It would be great to be able to create animations and export them.
